build-aux/.run-format.zsh

```diff
@@ -61,6 +61,24 @@ invoke_formatter() {
       local -a format_args=(-style=file -fallback-style=none)
       if (( _loglevel > 2 )) format_args+=(--verbose)
       ;;
+    gersemi)
```

Comment:
I'm leaving this comment here because I didn't go through to find the exact reason for that:
I've pulled this branch locally and I've noticed that `gersemi` is invoked with one file at a time. I'd recommend leveraging the fact that `gersemi` can check/reformat multiple paths with single invocation and it'll use subprocesses for parallelism. Unfortunately there's a relatively expensive initialization step related to preparing CMake parser which makes running gersemi once per file wasteful and significantly slower. If there's anything that you need from output point of view to leverage that just let me know through the issue on gersemi page.

Reply:
That's currently by design due to how the verbose output of the script works. The script wants to print a custom error message per file which requires formatting changes but without actually applying them (the files need to be untouched).

The way we achieve this right now is compare the formatted version printed to `stdout` to the actual file and check if it's different.

The `--check` parameter achieves the same, but we cannot customise its output (and that specific part of the script is formatter-agnostic).

Reply:
The script has been updated to allow `gersemi` to run for all changed files only once.